projects/marshamallowdefi/index.js

```diff
@@ -0,0 +1,40 @@
+const abi = require("./abi.json");
+const { transformBscAddress } = require("../helper/portedTokens");
+const { addFundsInMasterChef } = require("../helper/masterchef");
+
+const masterChefFarms = "0x8932a6265b01D1D4e1650fEB8Ac38f9D79D3957b";
+const masterChefSafeFarms = "0xEE49Aa34833Ca3b7d873ED63CDBc031A09226a5d";
+
+const bscTvl = async (chainBlocks) => {
+  const balances = {};
+
+  let transformAddress = await transformBscAddress();
+  await addFundsInMasterChef(
```

Comment:
is the address to ignore incorrect? there seems to be nothing at that address (https://bscscan.com/address/0x00000000548997391c670a5179Af731A30e7c3Ad)

Reply:
that's address has nothing, but it breaks the code, becuase takes the balance as null, that's the why I ignored it